solutions/account-infrastructure-base/variables.tf

```diff
@@ -55,6 +55,12 @@ variable "provision_atracker_cos" {
   default     = false
 }
 
+variable "cos_skip_iam_authorization_policy" {
+  type        = bool
+  description = "Whether to enable creating an IAM authoriation policy between the IBM Cloud Object Storage instance and the Key Management service instance of the CRN provided in `var.kms_key_crn`. This variable has no effect if `var.provision_atracker_cos` is false."
```

Comment:
dont use `var.` in DA variable descriptions. They don't make sense in Projects land :)